components/mpas-ocean/src/analysis_members/Registry_global_stats.xml

```diff
@@ -71,7 +71,7 @@
 				 description="Minimum global value of tangentialVelocity on ocean edges."
 			/>
 			<var name="layerThicknessEdgeMin" array_group="mins" units="m"
-				 description="Minimum global value of layerThicknessEdge on ocean edges."
+				 description="Minimum global value of layerThicknessEdgeFlux on ocean edges."
```

Comment:
I think this is a bit confusing.  I think we need probably either need `layerThicknessEdgeFluxMin`, etc. and `layerThicknessEdgeMeanMin` (also very confusing) or we need to just remove `layerTicknessEdge*` from the global stats.  @mark-petersen, do you have an opinion on this?

Reply:
I have added global stats for both `layerThicknessEdgeFlux` and `layerThicknessEdgeMean`.